src/markdown-pages/automate-workflows/get-started-terraform.mdx

```diff
@@ -37,273 +45,327 @@ To use this guide, you should have some basic knowledge of both New Relic and Te
+  policy_id       = newrelic_alert_policy.golden_signal_policy.id
+  name            = "High Error Percentage"
+  type            = "apm_app_metric"
+  entities        = [data.newrelic_entity.example_app.application_id]
+  metric          = "error_percentage"
+  runbook_url     = "https://www.example.com"
+  condition_scope = "application"
 
-    # Response time alert condition will not be provisioned unless a critical violation threshold is specified.
-    response_time_critical_threshold = 3
+  # Define a critical alert threshold that will trigger after 5 minutes above a 5% error rate.
+  critical {
+    duration      = 5
+    operator      = "above"
+    threshold     = "5"
+    time_function = "all"
+  }
+}
+```
 
-    channel_ids = [data.newrelic_alert_channel.pager.id]
+##### **Saturation**
+Saturation represents how "full" your service is and can take many forms, such as CPU time, memory allocation, or queue depth. In this example, let's assume you already have a [New Relic Infrastructure](https://newrelic.com/products/infrastructure) agent installed on the hosts serving your application and you want an alert configured for when CPU utilization spikes above a certain threshold.
+
+For Infrastructure alerts, we can use the [`newrelic_infra_alert_condition`](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/infra_alert_condition) resource. This alert will trigger if the aggregate CPU usage on these hosts rises above 90% for five minutes.
+
+```hcl
+# High CPU usage
+resource "newrelic_infra_alert_condition" "high_cpu" {
+  policy_id   = newrelic_alert_policy.golden_signal_policy.id
+  name        = "High CPU usage"
+  type        = "infra_metric"
+  event       = "SystemSample"
+  select      = "cpuPercent"
+  comparison  = "above"
+  runbook_url = "https://www.example.com"
+  where       = "(`applicationId` = '${data.newrelic_entity.example_app.application_id}')"
+
+  # Define a critical alert threshold that will trigger after 5 minutes above 90% CPU utilization.
+  critical {
+    duration      = 5
+    value         = 90
+    time_function = "all"
+  }
 }
 ```
+</Step>
+
+<!-- NOTIFICATION CHANNEL -->
+
+<Step>
+
+## Get notified when an alert is triggered
 
-</Step></Steps>
+Now that you have some important alert conditions configured, let's add a notification channel to your alert policy ensure the proper folks get notified when an alert is triggered. We'll use the [`newrelic_alert_channel`](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/alert_channel) resource, which [supports several types](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/alert_channel#argument-reference) of notification channels.
 
-## Tip: Avoid resources being marked as "sensitive"
+Since Slack is quite popular, let's create a Slack notification channel. When an alert is triggered, a notification will be sent to the Slack channel of your choice.
```

Comment:
I love showing that the user can set up an alert channel for Slack. However, because this example requires prerequisites, I wonder if it would be more effective to just use email. That way they can actually run this code without doing anything else and see the results right away.

Reply:
Yeah, I went back and forth on this. I also thought about including Slack _and_ email. I can add email and make the Slack portion a "bonus" option for the ambitious crowd. Thoughts on that?

Reply:
I think that's a great option!